dev-itpro/administration/administration-center-api_authorizedaadapps

```diff
@@ -0,0 +1,68 @@
+---
+title: Business Central Admin Center API - Authorized AAD Apps
```

Comment:
is the extension of this file correct? It does not display for me as rich text in review